### ISSUE TITLE
OMOP.episode (103.sql) : SQL Error: ORA-00942 table or view does not exist

### ISSUE THREAD

**Issue by scossin** [ohdsi-studies/PioneerWatchfulWaiting#60]
The errors occurs here: 

https://github.com/ohdsi-studies/PioneerWatchfulWaiting/blob/ca047952645cd576ab9bd1137bff0a3af3627511/inst/sql/sql_server/103.sql#L565

My sql is translated like this: 
```sql
--adding episode
    UNION all
  select ep.person_id, ep.episode_id, ep.episode_start_datetime, ep.episode_end_datetime,ep.episode_object_concept_id,null
  FROM OMOP.episode ep
```

The episode table is an OMOP extension: https://ohdsi.github.io/CommonDataModel/oncology.html
The SQL code seems to check if the table exists or not: 
https://github.com/ohdsi-studies/PioneerWatchfulWaiting/blob/ca047952645cd576ab9bd1137bff0a3af3627511/inst/sql/sql_server/103.sql#L563
But for some reason it failed to detect that I don't have this table.

**Comment by scossin** [comment]
As a temporary fix, I hardcoded episodetable=FALSE in CohortConstruction.R and it worked: 
https://github.com/scossin/PioneerWatchfulWaiting/commit/9359fb865a3d3da0cf27dda0030fd46454d365d1

**Comment by keesvanbochove** [comment]
This is fixed with #63.
